### PR TITLE
DP-996 - Create new DP examples for 6.0 in DP-Example Repo

### PR DESCRIPTION
**Pull request by gvjoshi25**

### CODE REVIEW

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/ReadUsingTimedReader.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import com.northconcepts.datapipeline.job.Job;
+
+public class ReadUsingTimedReader {
+
+	public static void main(String[] args) {
+		//Create a CardReader to retrieve records with specified delay in read.
+		DataReader reader = new CardReader()
+        		.setMaxRecords(10000)
+        		.setReadPeriod(TimeUnit.SECONDS.toMillis(1))
+				.setStartingDate("2016-01-14 09:00:00.000")
```
Comment (DeleTaylor) [inline src/main/java/com/northconcepts/datapipeline/examples/cookbook/ReadUsingTimedReader.java]: Can you fix the indentation by converting tabs to spaces?

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/SerializeAndDeserializeRecords.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.math.BigInteger;
+
+import com.northconcepts.datapipeline.core.Record;
+
+public class SerializeAndDeserializeRecords {
+
+	public static void main(String[] args) throws Throwable {
+		Record record1 = new Record();
```
Comment (DeleTaylor) [inline src/main/java/com/northconcepts/datapipeline/examples/cookbook/SerializeAndDeserializeRecords.java]: More indentation issues.

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/UpsertRecordsToPostgreSql.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    public static void main(String[] args) throws Throwable {
+        final String DATABASE_DRIVER = "org.postgresql.Driver";
+        final String DATABASE_URL = "jdbc:postgresql://localhost:5432";
+        final String DATABASE_USERNAME = "test";
+        final String DATABASE_PASSWORD = "test";
+        final String DATABASE_TABLE = "CreditBalance";
+        final String PRIMARY_KEY = "Account";
+
+        Class.forName(DATABASE_DRIVER);
+		Connection connection = DriverManager.getConnection(DATABASE_URL, DATABASE_USERNAME, DATABASE_PASSWORD);
```
Comment (DeleTaylor) [inline src/main/java/com/northconcepts/datapipeline/examples/cookbook/UpsertRecordsToPostgreSql.java]: More indentation issues.

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/WriteBigDecimalAndBigIntegerToExcel.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        record2.setField("creditLimit", new BigInteger("12345678901234567890"));
+        
+        DataReader reader = new MemoryReader(new RecordList(record1, record2));
+        
+		ExcelDocument document = new ExcelDocument();
+		DataWriter writer = new ExcelWriter(document).setSheetName("balance");
+
+		Job.run(reader, writer);
+
+		document.save(new File("example/data/output/WriteBigDecimalAndBigIntegerToExcel-credit-balance.xls"));
```
Comment (DeleTaylor) [inline src/main/java/com/northconcepts/datapipeline/examples/cookbook/WriteBigDecimalAndBigIntegerToExcel.java]: Fix indentation.

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/customization/BlacklistAndWhitelistFunctionsInExpressionLanguage.java [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+	}
+	
+	private static void blacklistingCustomFunctions() {
+		System.out.println("\n\n================================Blacklisting Custom Functions================================--");
+		
+		try {
+			DataReader reader = new CSVReader(new File("example/data/input/credit-balance-01.csv"))
+					.setFieldNamesInFirstRow(true);
+	
+			TransformingReader transformingReader = new TransformingReader(reader);
```
Comment (DeleTaylor) [inline src/main/java/com/northconcepts/datapipeline/examples/cookbook/customization/BlacklistAndWhitelistFunctionsInExpressionLanguage.java]: Does this need to be declared here or can we move is down to where it's used?

**Review by DeleTaylor — CHANGES_REQUESTED** [review]

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/ReadUsingTimedReader.java (hunk 1 above)
Comment (gvjoshi25) [reply src/main/java/com/northconcepts/datapipeline/examples/cookbook/ReadUsingTimedReader.java]: converted indentation to spaces and fixed it.

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/SerializeAndDeserializeRecords.java (hunk 2 above)
Comment (gvjoshi25) [reply src/main/java/com/northconcepts/datapipeline/examples/cookbook/SerializeAndDeserializeRecords.java]: converted indentation to spaces and fixed it.

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/UpsertRecordsToPostgreSql.java (hunk 3 above)
Comment (gvjoshi25) [reply src/main/java/com/northconcepts/datapipeline/examples/cookbook/UpsertRecordsToPostgreSql.java]: converted indentation to spaces and fixed it.

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/WriteBigDecimalAndBigIntegerToExcel.java (hunk 4 above)
Comment (gvjoshi25) [reply src/main/java/com/northconcepts/datapipeline/examples/cookbook/WriteBigDecimalAndBigIntegerToExcel.java]: converted indentation to spaces and fixed it.

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/customization/BlacklistAndWhitelistFunctionsInExpressionLanguage.java (hunk 5 above)
Comment (gvjoshi25) [reply src/main/java/com/northconcepts/datapipeline/examples/cookbook/customization/BlacklistAndWhitelistFunctionsInExpressionLanguage.java]: Yes, right, it should defined just before it is being used. I have fixed it.

**Review by DeleTaylor — APPROVED** [review]

**Pull request merged**